ajax/libs/slim.js/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "slim.js",
+  "filename": "slim.js",
```

Comment:
Would you like to provide the minified `.js` file?
If not, it's fine. 
CDNJS would provide it with [uglify-js](https://www.npmjs.com/package/uglify-js), so please modify to "filename": "Slim.min.js".